guides/common/modules/proc_providing-feedback-on-red-hat-documentation.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: PROCEDURE
+
+[preface]
+[id="proc_providing-feedback-on-red-hat-documentation_{context}"]
```

Comment:
Hi @maximiliankolb , thank you very much for your suggestions for improvement. I fully understand them. However, in this case, I would like to ask if it would be acceptable to skip them all, please? And merge the module as it is? The purpose of the new module is to inform the users of *downstream* documentation how to provide the feedback on documentation. There are two ways - either the DDF (Direct Documentation Feedback) - they just highlight a piece of text, the "feedback" button appears, they type their feedback, and by submitting it, a Bugzilla ticket is opened automatically. The second way is to open a Bugzilla ticket manually. The module I'm adding is to explain these two ways. The module exactly as it is, is used in all RHEL guides and maybe also in some other RH products. We would like to use exactly the same for RH Satellite. It was not needed before, because the DDF  functionality was not enabled for previous versions of RH Satellite docs. It has been enabled with Satellite 6.11 (it was a long-term plan), so we need to add that module into all guides that are published for RH Satellite. Please, understand the situation that if some Satellite guides are based on upstream repo and some on downstream, it might look quite inconsistent if we use different modules. The explanatory module on how to provide feedback must look the same in all guides. The module is only applicable to Satellite, so it will never appear in upstream documentation. So please could it be accepted as it is? As to the cherry-picking, I thought that if it's needed only for versions relevant to Satellite release, then we don't need to request cherry-picking to 3.2. But if you think it's a safer way to cherry-pick it everywhere, than I don't have any concerns about it and let's cherry-pick it to 3.2 as well. Apologies for any inconvenience. I really appreciate  all your suggestions for improvement, which I would incorporate in a normal case, but please understand that this is a bit exceptional situation. Also, we need to get the instructions on how to use the DDF tool to customers ASAP. Thank you!

Reply:
Hi @mdolezelova 

I did not request a total rework/rewrite, but trivial changes to align your proposed change to our community standards. For more information, see CONTRIBUTING.md or seek help internally.

Regarding cherry-picking: It most likely makes things easier for everyone if we cherry-pick to all versions between 'master' and '3.1' as well.